corehq/apps/locations/views.py

```diff
@@ -834,6 +834,16 @@ def location_export(request, domain):
     return response
 
 
+def get_locations_paylod_from_ids(ids, domain, base_queryset):
+    from corehq.apps.locations.util import get_locations_from_ids
+    locations = get_locations_from_ids(ids, domain, base_queryset=base_queryset)
+    return [loc_to_payload(loc) for loc in locations]
+
+
+def base_queryset_for_domain_and_user(domain, user):
+    return SQLLocation.objects.accessible_to_user(domain, user)
```

Comment:
@esoergel I think for admin user we would still need to keep `self.all()` since they don't have domain membership and have access to all domain. Will update using that.